frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-list/experiment-list.component.ts

```diff
@@ -172,6 +172,7 @@ export class ExperimentListComponent implements OnInit, OnDestroy, AfterViewInit
   }
 
   filterExperimentByChips(tagValue: FLAG_SEARCH_SORT_KEY, type: EXPERIMENT_SEARCH_KEY) {
+    this.searchValue = tagValue;
```

Comment:
This is not a proper solution because you already have a subscriber for getting the searchValue but here you are setting the value directly.